template/src/components/header/AccountInfoPanel.tsx

```diff
@@ -16,20 +16,14 @@ export function AccountInfoPanel() {
       <div className="my-4 inline-flex items-center justify-start gap-2">
         <OnchainAvatar address={address} className="h-10 w-10 rounded-full" />
         <div className="inline-flex flex-col items-start justify-center gap-1">
-          <div className="inline-flex items-center justify-start gap-1">
-            <div className="font-inter w-32 text-base font-medium text-white">{ensName}</div>
-          </div>
-          <div className="inline-flex items-center justify-start gap-8">
-            <div className="flex items-center justify-start gap-1">
-              <OnchainAddress
-                address={address}
-                className="font-inter w-32 text-sm font-medium text-zinc-400"
-              />
-            </div>
-          </div>
+          <div className="font-inter w-32 text-base font-medium text-white">{ensName}</div>
+          <OnchainAddress
+            address={address}
+            className="font-inter w-32 text-sm font-medium text-zinc-400"
+          />
         </div>
       </div>
-      <div className="h-px self-stretch bg-zinc-400 bg-opacity-20" />
+      <hr className="h-px self-stretch bg-zinc-400 bg-opacity-20" />
```

Comment:
Why the change from div to hr? Would prefer a div over an hr.

Reply:
In this case, this element is supposed to render a separator between the onchain address and the logout button

Having this be an hr element instead of a seems to fit better from a Semantic HTML POV, and the feedback from the earlier PR was to use more Semantic HTML

Reply:
Link to the feedback: https://github.com/coinbase/build-onchain-apps/pull/329#discussion_r1481025596